src/main/java/org/mdbenefits/app/preparers/ApplicantDetailsPreparer.java

```diff
@@ -34,7 +34,7 @@ public Map<String, SubmissionField> prepareSubmissionFields(Submission submissio
         results.put("applicantDOB", new SingleField("applicantDOB", (String) dob, null));
 
         results.put("applicantSSN",
-                new SingleField("applicantSSN", SubmissionUtilities.formatSSN((String) inputData.get("encryptedSSN")), null));
+                new SingleField("applicantSSN", SubmissionUtilities.formatSSN((String) inputData.get("applicantSSN")), null));
```

Comment:
Fixes the same issue from this ticket:
https://www.pivotaltracker.com/story/show/187040206

Encryption will be handled later in this ticket:
https://www.pivotaltracker.com/n/projects/2688497/stories/186997138